target_chains/solana/program_simulator/src/lib.rs

```diff
@@ -49,9 +57,13 @@ impl ProgramSimulator {
         signers: &Vec<&Keypair>,
         payer: Option<&Keypair>,
     ) -> Result<(), BanksClientError> {
+        let compute_units_ixs =
+            compute_budget::ComputeBudgetInstruction::set_compute_unit_limit(2000000);
```

Comment:
Adding the compute unit limit here may surprise callers of this method. Suggest making the limit an argument to the function so it's explicit

Reply:
or call the method `process_ix_with_default_compute_limit`

Reply:
Renamed